### ISSUE TITLE
Browse by podcast_episode - default sort order

### ISSUE THREAD

**Issue by rsinfotech** [ampache/ampache#3320]
I'm getting an odd default sort order of the results when browsing the GUI with browse.php?action=podcast_episode

Don't know how it's sorting but it's not by Publication Date as I'd expect:

![image](https://user-images.githubusercontent.com/3216809/185768245-50dc2501-e2fe-4437-b5f6-fcca9ba6fc4c.png)

I have to click twice on the Publication Date column header to sort first in ascending order and again for descending order to see the latest downloaded podcasts first in the list.

Should this be the default behaviour?

**Comment by lachlan-00** [comment]
it actually doesn't have any sorting on the browse at all by default

**Comment by lachlan-00** [comment]
it's meant to be pubdate but i don't see any sql
```
get_sql: SELECT `podcast_episode`.`id` FROM `podcast_episode`   LIMIT 0, 50
```

**Comment by lachlan-00** [comment]
fixed with e63e10a52 made those sorts all lower

**Comment by rsinfotech** [comment]
Fabulous! :+1: Now sorting with latest podcasts first in the list, much appreciated.
